cmd/serve.go

```diff
@@ -70,6 +82,15 @@ func NewServeCmd(opts *ServeCmdOptions) *cobra.Command {
 				log.Fatal(err)
 			}
 
+			if len(opts.Domain) != 0 {
+				if len(opts.CertfilePath) == 0 || len(opts.KeyfilePath) == 0 {
+					opts.CertfilePath, opts.KeyfilePath = fmtLetsEncrypt(opts.Domain)
+				}
+				if err := srv.ListenAndServeTLS(opts.Domain+":443", opts.CertfilePath, opts.KeyfilePath); err != nil && err != http.ErrServerClosed {
+					log.Fatalf("listen: %s\n", err)
+				}
+			}
+
```

Comment:
I just noticed we don't have a addr flag to listen to a different address. I think it's worth adding a new `--addr` flag and use it for both TLS and non-TLS.

`--addr` should be empty by default and we should use TLS when `--cert` or `--key` is not empty.

In the [Gin's server implementation](https://github.com/gin-gonic/gin/blob/b04917c53e310e3746ec90cd93106cda8c956211/ginS/gins.go#L137-L146), it's the same option for both methods.